erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -7,9 +7,11 @@
 
 namespace erizo {
 
+const int ExternalOutput::PLI_INTERVAL = 5000;
```

Comment:
PLI is Picture Loss Indication.  For some more info check out:
https://svn.tools.ietf.org/html/draft-ietf-avt-avpf-ccm-10#section-2.1